systems/statuses.md

```diff
@@ -1,41 +1,124 @@
+
+ - A grappled character may not move using their move or manouvering actions.
+ - [Attacks](rolls.md#Attack) made against a grappled target are made with an [advantage](rolls.md#Advantage).
+
 The grappled character may use its action to attempt to escape the grapple using a strength test against the grappler.
 
-Attacks made against a grappled target are made with an [advantage](rolls.md#Advantage).
 
 ## Restrained
 Restrained is an advanced form of [grappled](#grappled). You may not be both grapped and restrained.
 
-The restrained character may not move or use any actions that require their hands, such as attacking, or casting spells with a somatic component.
-
-Attacks made against a restrained target are made with a `2x` [advantage](rolls.md#Advantage).
+ - A restrained character may not move using their move or manouvering actions.
+ - The restrained character may not use any actions that require their hands, such as [attacking](rolls.md#Attack), or casting [spells](spells.md) with a [somatic component](spells.md#Somatic).
+ - [Attacks](rolls.md#Attack) made against a restrained target are made with a `2x` [advantage](rolls.md#Advantage).
 
 The restrained character may use its action to attempt to escape being restrained using a strength test against the restrainer.
 
 ## Prone
-A prone character is lying on the ground. Their ability to move is reduced, and it is difficult to use weapons.
+A prone character is lying on the ground. Their ability to move is reduced, and it is difficult to use weapons. Your character may willingly become prone by taking half of their movement during a move action to do so.
+
+ - A prone character has its move actions modified. They may instead:
+    - [Move](actions.md#Move) action to move up to half their speed
+    - [Manouver](actions.md#Manouvering) action to stand up
+
+ - Any [melee attacks](rolls.md#Melee-attack) against a prone character are made with an [advantage](rolls.md#Advantage).
+ - Any [ranged attacks](rolls.md#Ranged-attack) against a prone character are made with a [disadvantage](rolls.md#Disadvantage).
 
-A prone character has its move actions modified. They may instead:
- - [Move](actions.md#Move) action to move up to half their speed
- - [Manouver](actions.md#Manouvering) action to stand up
+## Marked
+A marked character has been specially marked by some effect that makes them easy.
 
-Any melee attacks against a prone character are made with an [advantage](rolls.md#Advantage).
-Any ranged attacks against a prone character are made with a [disadvantage](rolls.md#Disadvantage).
+ - A marked character may not become [hidden](#Hidden)
+ - All [attacks](rolls.md#ttack) against a marked character are made with an [advantage](rolls.md#Advantage).
+
+## Blinded
+A blinded foe has lost the ability to see their foes.
+
+ - All foes are considered [hidden](#Hidden).

```

Comment:
Hmm, I wonder if this shouldn't be foe related, maybe instead something like `All creatures are considered hidden`. I wonder if there should be a distinction between `hidden` and `invisible` as well. You might not be able to see an enemy, but might know where they are. I feel hidden is undetected, and invisible is particularly the sight component. We could have the same sort of thing for deafened and not sound detected (inaudible?). You could have something on invisible which is like `A creature that is both invisible and inaudible is [hidden]` and such.

Reply:
I agree  - but I just dont want to have to define all the rules for becoming hidden right now.
I figured the invisible status would grant hidden on the condition that no other senses detect the character.

Its hard not to have a lot of rules around that.

Reply:
Wherever possible rules are worded character vs foe. I think its the 99% use case. The DM can file down the edges.